.circleci/config.yml

```diff
@@ -390,7 +390,15 @@ jobs:
             command: |
               bash -c "$PACKCHECK $BUILD" || exit 1
               count=$(find . -name "*.hs" -exec grep -H '\ $' {} \; | tee /dev/tty | wc -l)
-              exit $count
+              if [ $count > 0 ]
+              then exit 1
+              fi
+              git diff > tmp_len
+              count2=$(awk '{ if (length($0) > 80) {print "Line length exceeds 80 : " length($0); print $0; } }' tmp_len | tee /dev/tty | wc -l)
```

Comment:
The git diff output has deleted lines, changed lines and other info. We need to grep only the lines with a leading `+`  (`grep '^+') and filter out the lines with a leading `+++` (grep -v `^+++`) from that.

Reply:
Ah, yes!